exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+    return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+    return arr[arr.length - 1]; // complete this statement
```

Comment:
Good - you have correctly worked out that you need to use -1 as the array starts at 0.